src/pages/MentorshipPage.jsx

```diff
@@ -0,0 +1,141 @@
+import PlaceholderPic from '../assets/placeholderPic.png';
+import YellowPlaceholder from '../assets/yellowPlaceholder.png';
+import Profile from '../assets/Profile.png';
+import Footer from "../components/Footer";
+
+
+export function MentorshipPage() {
+    return (
+        <div className="flex flex-col items-center justify-center w-full min-h-screen bg-white p-8">
```

Comment:
its allowing me to move the page horizontally which is something we don't want it could be something in one of the overall div tags like this one